ledger/participant-integration-api/src/main/scala/platform/apiserver/services/tracking/QueueBackedTracker.scala

```diff
@@ -91,9 +91,13 @@ private[services] final class QueueBackedTracker(
 
   override def close(): Unit = {
     logger.debug("Shutting down tracking component.")
-    queue.complete()
-    Await.result(done, 30.seconds)
-    ()
+    if (done.isCompleted)
+      logger.info("Tracker already complete")
+    else {
+      queue.complete()
+      Await.result(done, 30.seconds).discard
+      ()
```

Comment:
```suggestion
```

Reply:
I tested the code in canton before copying it here, hence the `.discard` cropping in. 
We actually do need the `()`, lest scalac complains:
```
 warning: discarded non-Unit value of type akka.Done
```
We don't need the `discard` though.